addons/mail/static/src/components/message_reaction_group/message_reaction_group.xml

```diff
@@ -2,7 +2,11 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.MessageReactionGroup" owl="1">
-        <div class="o_MessageReactionGroup d-flex" t-att-class="{'o-hasUserReacted': messageReactionGroup and messageReactionGroup.hasUserReacted}" t-att-title="messageReactionGroup and messageReactionGroup.summary" t-on-click="messageReactionGroup and messageReactionGroup.onClick">
+        <div class="o_MessageReactionGroup d-flex"
+             t-att-class="{'o-hasUserReacted': messageReactionGroup and messageReactionGroup.hasUserReacted}"
+             t-att-title="messageReactionGroup and messageReactionGroup.summary"
+             t-on-click="messageReactionGroup and messageReactionGroup.messageReactionGroupViews[0].onClick"
```

Comment:
The component `MessageReactionGroupSummaryView` should have props local id and getter for record with model `MesssageReactionGroupView`, instead of record with model `MessageReactionGroup`.